api/lib/domain/usecases/complete-assessment.js

```diff
@@ -24,27 +28,58 @@ module.exports = async function completeAssessment({
   assessment.setCompleted();
 
   if (assessment.isCertification()) {
-    await _calculateCertificationScore({ assessment, assessmentResultRepository, certificationCourseRepository, competenceMarkRepository, scoringCertificationService });
+    await _calculateCertificationScore({
+      assessment,
+      assessmentResultRepository,
+      certificationCourseRepository,
+      competenceMarkRepository,
+      badgeAcquisitionRepository,
+      certificationPartnerAcquisitionRepository,
+      scoringCertificationService
+    });
   }
   await assessmentRepository.completeByAssessmentId(assessmentId);
   return assessment;
 };
 
+function _checkCriteriaFullfilledClea(hasBadgeClea, percentageCorrectAnswers) {
+  if (hasBadgeClea) {
+    // Green zone
+    return percentageCorrectAnswers >= 80;
+  }
+  return false;
+}
+
 async function _calculateCertificationScore({
   assessment,
   assessmentResultRepository,
   certificationCourseRepository,
   competenceMarkRepository,
+  badgeAcquisitionRepository,
+  certificationPartnerAcquisitionRepository,
   scoringCertificationService,
 }) {
   try {
     const assessmentScore = await scoringCertificationService.calculateAssessmentScore(assessment);
+
+    const partnerCertifications = [];
+    if (await badgeAcquisitionRepository.hasAcquiredBadge({
+      badgeKey: Badge.keys.CLEA,
+      userId: assessment.userId
+    })) {
+      if (_checkCriteriaFullfilledClea(true, assessmentScore.percentageCorrectAnswers)) {
```

Comment:
C'est un peu bizarre`true` comme premier paramètre non?
```suggestion
    const hasBadge = await badgeAcquisitionRepository.hasAcquiredBadge({
      badgeKey: Badge.keys.CLEA,
      userId: assessment.userId
    });
      if (hasBadge && _checkCriteriaFullfilledClea(hasBadge, assessmentScore.percentageCorrectAnswers)) {
```

Reply:
Quoi que je ne sais pas si `hasBadge &&` est nécessaire

Reply:
j'ai modifié mon workflow et du coup cette condition n'est plus très pertinente effectivement

Reply:
corrigé